File: 1-two-sum/two-sum.go

func twoSum(nums []int, target int) []int {
	ans := []int{}
	for i := 0; i < len(nums); i++ {
		for j := 0; j < len(nums); j++ {
			tmp := nums[i] + nums[j]
			if i != j && tmp == target {
				ans = []int{i, j}
				break
			}
		}
		if len(ans) > 0 {
			break
		}
	}
	return ans
}

File: 1-two-sum/two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:

        for i in range(len(nums)):
            tmp = target - nums[i]
            if tmp in nums:
                j = nums.index(tmp)
                if i != j:
                    return [i, j]